spec/lib/aamva/proofer_spec.rb

```diff
@@ -55,26 +55,28 @@
     end
 
     context 'when verification is unsuccessful' do
-      let(:success) { false }
-      let(:verification_results) { super().merge(dob: false, zipcode: false) }
```

Comment:
when I updated this test to stop stubbing intermediate objects and just stub the network request, I removed the zipcode references, zipcode/postal code appears to have been removed in https://github.com/18F/identity-aamva-api-client-gem/pull/18